src/components/Post.tsx

```diff
@@ -21,24 +19,75 @@ import Animated, {
   useSharedValue,
   withTiming,
 } from 'react-native-reanimated';
+import Video, {VideoRef} from 'react-native-video';
+import Slider from '@react-native-community/slider';
+import DescriptionView from './DescriptionView';
+import {formatProgressTime} from '../util/ProgressTimeFormatter';
 
 interface PostProps {
   item: PostType;
+  activePostId: number;
 }
 
-export default function Post({item}: PostProps) {
-  const {height} = useWindowDimensions();
-  const maxHeroHeight = height - (64 + 60 + 40 + 60 + 10); // right under the header
+export default function Post({item, activePostId}: PostProps) {
   const [isLiked, setIsLiked] = useState(false);
   const [isShowMore, setIsShowMore] = useState(false);
   const [contentHeight, setContentHeight] = useState(0);
+  const [ended, setEnded] = useState(false);
+  const [currentTime, setCurrentTime] = useState<number>();
+  const [duration, setDuration] = useState<number>();
+  const [isPlaying, setIsPlaying] = useState(false);
```

Comment:
why do we still have both isPlaying & paused? this should be removed and rely only on paused state